src/modeling_library/distributions/binom.jl

```diff
@@ -0,0 +1,29 @@
+struct Binomial <: Distribution{Int} end
+
+"""
+	binom(n::Integer, p::Real)
+Sample an `Int` from the Binomial distribution with parameters `n` (number of
+trials) and `p` (probability of success in each trial).
+"""
+const binom = Binomial()
+
+function logpdf(::Binomial, x::Integer, n::Integer, p::Real)
+	if x < 0 return -Inf end
+	coefficient = loggamma(n + 1) - loggamma(n + 1 - x) - loggamma(x + 1)
+	coefficient + x * log(p) + (n - x) * log(1 - p)
```

Comment:
```suggestion
	return coefficient + x * log(p) + (n - x) * log(1 - p)
```

Reply:
This isn't documented anywhere, and is done inconsistently now, but I think it would be good to add explicit return statements to multi-line function definitions, as suggested in this style guide: https://github.com/jrevels/YASGuide#programming-guidelines (and I have noticed you have generally being doing this elsewhere, and I agree it is good style).